src/helpers/ignoreFilesPattern.ts

```diff
@@ -6,7 +6,7 @@ import type { Site } from '../types';
 
 export const excludePatterns = ['conf'];
 
-export const localBackupsIgnoreFileName = '.localbackupaddonignore';
+export const localBackupsIgnoreFileName = '.localbackupaddonignore.txt';
```

Comment:
This willl probably break existing instances of the file. (i.e. a new file will be created since the name is being changed and any values in the previous file that may be user added wouldn't be present). One way to get around this could be to write some migration logic that runs when backups are created or restored to check for the the old filename and replace with the new.

Does this work for Windows/Linux with their equivalent default text editor apps?

Reply:
This might be a good place for the "migration" logic :thinking: 
https://github.com/getflywheel/local-addon-backups/blob/master/src/helpers/ignoreFilesPattern.ts#L26

Reply:
Good call out, I update the logic to be more backwards compatible. Thanks for the review!

Reply:
Dope!